docs-src/content/getting-started/streaming.md

```diff
@@ -101,12 +107,213 @@ audio = open('test.wav', 'rb')
 # print out results as they come in
 for resp in client.StreamingRecognize(cfg, audio):
     for result in resp.results:
-		if result.is_partial:
-			print("\r{0}".format(result.alternatives[0].transcript), end="")
-		else:
-			print("\r{0}".format(result.alternatives[0].transcript), end="\n")
+        if result.is_partial:
+            print("\r{0}".format(result.alternatives[0].transcript), end="")
+        else:
+            print("\r{0}".format(result.alternatives[0].transcript), end="\n")
+
+```
+{{% /tab %}}
+
+{{%/tabs %}}
+
+### Streaming from microphone
+
+The examples require an external library to talk to the sound driver,
+in order to stream audio from the microphone. Several libraries are
+available but the recommended ones are mentioned for each language below.
+
+{{%tabs %}}
+{{% tab "Go" %}}
+
+This example utilizes the [portaudio](https://github.com/gordonklaus/portaudio) bindings for Go
+to stream audio from a microphone. To use this package, install [PortAudio](http://www.portaudio.com/)
+development headers and libraries using an appropriate package manager for your system (e.g. `apt-get install portaudio19-dev` on Ubuntu, `brew install portaudio` for OSX, etc.) or build from [source](http://portaudio.com/docs/v19-doxydocs/tutorial_start.html).
+
+``` go
+package main
+
+import (
+    "bytes"
+    "context"
+    "encoding/binary"
+    "fmt"
+    "log"
+
+    "github.com/cobaltspeech/sdk-cubic/grpc/go-cubic"
+    "github.com/cobaltspeech/sdk-cubic/grpc/go-cubic/cubicpb"
+    "github.com/gordonklaus/portaudio"
+)
+
+const serverAddr = "127.0.0.1:2727"
+
+// Microphone object which can be passed to Cubic's streaming
```

Comment:
Microphone implements the io.ReadCloser interface and provides a data stream for microphone input.